Zend/zend_language_parser.y

```diff
@@ -701,8 +701,8 @@ class_statement_list:
 class_statement:
 		variable_modifiers property_list ';'
 			{ $$ = $2; $$->attr = $1; }
-	|	T_CONST class_const_list ';'
-			{ $$ = $2; RESET_DOC_COMMENT(); }
+	|	method_modifiers T_CONST class_const_list ';'
+			{ $$ = $3; $$->attr = $1; RESET_DOC_COMMENT(); }
```

Comment:
Right. I'll take care about them.

I can only trust you about RESET_DOC_COMMENT() :)